docs/source/dummy_table.rst

```diff
@@ -0,0 +1,143 @@
+.. _dummy_table:
+
+Dummy Table
+###########
+
+Same old classification table let's see.
+
+.. rst-class:: table-weights
+.. table::
+    :class: datatable
```

Comment:
The way to set the css class of our tables is to use `.. rst-class: ` as above. We set that here:

https://github.com/pytorch/vision/blob/3ec4b94998bd2b413f1a5c77bfed590ce684b9b9/docs/source/conf.py#L379:L379

I ak garbage at CSS and web dev in general. Do you know if stuff can have 2 different classes?

Reply:
Yeah why not stuff can have as many CSS classes as possible. There is no limit. Only thing is classes are overridden according to CSS class hierarchy. (inline css over style tag over DOM selector).

Reply:
OK, then I assume we can try updating the line to

```
table_file.write(".. rst-class:: table-weights datatable\n")
```

(not sure about the order)

Reply:
Yeah don't worry i will take care :smile: